arch/Kconfig

```
@@ -434,6 +434,12 @@ config ARCH_HAVE_FORK
 	bool
 	default n
 
+config ARCH_HAVE_CRC32
+	bool
+	default n
+	---help---
+		Decide whether support CRC32 instruction
```

```suggestion
		Architecture supports CRC32 instruction
```
This might be better. Thanks :-)